Formula/nb.rb

```diff
@@ -0,0 +1,39 @@
+class Nb < Formula
+  desc "Command-line and local web note‑taking, bookmarking, and archiving"
+  homepage "https://xwmx.github.io/nb"
+  url "https://github.com/xwmx/nb.git",
+      tag:      "6.11.1",
+      revision: "f3b2b6757f523c25d67d8e4f1cde6e376960a324"
```

Comment:
Do we need a clone of the Git repo? Can we use a tarball?

Reply:
The `nb` repository doesn't have releases and only tags commits. If GitHub provides the ability to download tarballs of specific git tags, it's possible to use a tarball, otherwise, we can't.

Reply:
yeah, I think that is we preferred, instead of using the git-checkout, we prefer the tarball file because it would be a small footprint for the build